content/en/blog/_posts/2021-10-05-nsa-cisa-hardening.md

```diff
@@ -29,6 +29,18 @@ _Note_: This blog post is not a substitute for reading the guide. Reading the pu
 guidance is recommended before proceeding as the following content is 
 complementary.
 
+**Update, November 2023**  
+_The National Security Agency (NSA) and the Cybersecurity and Infrastructure Security Agency (CISA)_  
+_released the 1.0 version of the Kubernetes hardening guide in August 2021 and updated it based on_  
+_industry feedback in March 2022 (version 1.1)._  
+_The most recent version of the Kubernetes hardening guidance was released in August 2022_  
+_with corrections and clarifications._  
+_Version 1.2 outlines a number of recommendations for_  
+[_hardening Kubernetes clusters_](https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF).
```

Comment:
Hmm, I didn't expect this to wrap like it has. Please amend so it renders as a paragraph.

Reply:
ok I change it to paragraph without italic font